Package.swift

```diff
@@ -37,12 +37,28 @@ let package = Package(
   targets: [
     .target(
         name: "SwiftProtobuf",
-        exclude: ["CMakeLists.txt"]
-    ),
+        exclude: ["CMakeLists.txt"],
+				swiftSettings: [
+					/// giving you a nice red compile error in Xcode on the spot if you try to use some unsafe API
+					.unsafeFlags(["-Xfrontend", "-application-extension"])
```

Comment:
We can't add those flags sadly because the use of `unsafeFlags` makes packages not consumable by upstream packages/applications anymore. There are ongoing discussions to make some of the unsafe flags actually safe to use by elevating them to concrete options in the package description. cc @neonichu

Reply:
> We can't add those flags sadly because the use of `unsafeFlags` makes packages not consumable by upstream packages/applications anymore.

Sounds like this means we should go ahead and close this since we'd immediately hit problems with Swift gRPC using this and then folks trying to use that?

Reply:
Yes, I think we can't go forward with this PR. I would recommend @tkafka that you file a Feedback in the Apple feedback system.

Reply:
@thomasvl What kind of problems, please? It seems to me that 
1. we just tell xcode, that `swift-protobuf` isn't using some APIs (while the library users, like `Swift gRPC`, can still call them just like before)
2. and we extend the potential uses of `swift-protobuf` to application extensions (again, not changing anything for the existing users)

I am not sure what should I file the feedback to Apple about - that they added flags to let package users know that it is using a safe subset of APIs, but that we refuse to use this flag (even though it seems we safely use that subset of APIs)?